contracts/MemeFactory.sol

```diff
@@ -388,7 +388,7 @@ abstract contract MemeFactory {
         _locked = 2;
 
         // Check for name and symbol lengths
-        require(bytes(name).length > 0 && bytes(name).length > 0, "Name and symbol must not be empty");
+        require(bytes(name).length > 0 && bytes(symbol).length > 0, "Name and symbol must not be empty");
```

Comment:
Lol - I didn't even see the second issue until I had a failing test for it. NEED MORE TESTS!